chroot.sh

```diff
@@ -22,7 +22,7 @@ EOF
 apt update
 apt install linux-image-6.1.0-20-amd64 network-manager console-setup console-setup-linux pciutils \
     xserver-xorg-video-all xserver-xorg-input-evdev x11-xserver-utils \
-    x11-xkb-utils x11-utils xinit chromium openbox wmctrl neovim git lightdm sudo xdotool -y
+    x11-xkb-utils x11-utils xinit chromium openbox wmctrl neovim git lightdm sudo xdotool tar curl -y
```

Comment:
I added these so like u can download the gotty file with curl and unzip the file tar. and yes its not available in apt packagemanager